src/store/getters.ts

```diff
@@ -9,3 +32,355 @@ export const getNameRequestUrl = (state: any): string => {
 export const isAuthenticated = (state: any): boolean => {
   return Boolean(sessionStorage.getItem(SessionStorageKeys.KeyCloakToken))
 }
+
+/** Returns the name. */
```

Comment:
Semantic, but these aren't functions that return something. They're getters, and appear like properties :)

Reply:
i see :) So should we avoid the doc-block comments or rephrase to `The Name` or `The Name Search` etc?

Reply:
In my IDE, hovering over these getters shows me the JsDoc stuff, so it's still valuable.

Yeah, I'd go with simply "The name." (but the name of what?)

Reply:
PS Documenting the return type is redundant. Sometimes I'll use the @returns tag to give more context.